### ISSUE TITLE
synchronization does not show a log when using the "show more details" option

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#1354]
Though synchronization seems to have run normally, I was unable to see the complete logs even when clicking on the "show more details" option. Not sure if that is related to this, either.

@luizaogs got it and I got it, too, in the test environment.

_Originally posted by @urapadmin in https://github.com/arch-kiosk/kiosk/issues/1350#issuecomment-1037003533_

This is presumably due to:
- [x] #1368

**Comment by urapadmin** [comment]
### Kiosk v0.15.
  - I hope this works, too, if #1368 is fixed

**Comment by luizaogs** [comment]
It does.

**Comment by urapadmin** [comment]
hm. This happened again with the new PACHA server. I must repoen this.

**Comment by urapadmin** [comment]
this cannot be due to the issue in #1368 with early cancellation. It must be something else. Perhaps because the PACHA DB was totally empty?

**Comment by urapadmin** [comment]
yes, happens on the new demo server, too. The logs show properly when cancelling a workstation process. So it seems to be limited to synchronization.

**Comment by luizaogs** [comment]
This is happening in PASU and the database is no longer totally empty, fyi.

**Comment by urapadmin** [comment]
### kiosk 0.25.5
The javascript code around this is pretty convoluted and embarrassing. But changing it bears danger of too many side effects so I have decided to just hide the "show details" button if there are no warnings. 
If there are warnings indeed, everything should still be fine - as it has been.

**Comment by luizaogs** [comment]
Lutz, could you release 0.25.5 on PACHA? I just tried testing this there before realizing that it's not yet on that version. PVD always has a million warnings so I can't actually see if this is working properly there.

**Comment by urapadmin** [comment]
tomorrow :) I am sleepy.

**Comment by luizaogs** [comment]
I didn't mean immediately :)
